### DISCUSSION TITLE
How to make buttons to close and minimize windows on the right?

### DISCUSSION THREAD

**Discussion by blackcement** [elementary/triage#504] [Q&A] [answered]
Lord, people, what planet are you from? Why are all the buttons flipped to the left? This is both in Explorer and in the browser. How to do this like in windows?

**Comment by teamcons** [comment, marked as answer]
Firstly, hello
Asking people "what planet they are from" may come off as aggressive. We do not owe you anything.

As for the second question - not all of them are. It is a design choice. Just like windows putting those on the right. The Elementary team does their thing, just like any other team offering their thing.

To answer your question, you can
-Install elementary-tweaks. It is one of the options this additional settings panel provides.
-Install gnome-tweaks. As Elementary is built upon GTK, the option it exposes should work
-Rely on solutions to change buttons placements in GNOME, theres tons of people asking for support for just that if you search online
